uitest/Pages/DashboardPage.cs

```diff
@@ -0,0 +1,94 @@
+﻿// Copyright (c) Microsoft Corporation and Contributors
+// Licensed under the MIT license.
+
+using DevHome.UITest.Dialogs;
+using DevHome.UITest.Extensions;
+using OpenQA.Selenium.Appium;
+using OpenQA.Selenium.Appium.Windows;
+
+namespace DevHome.UITest.Pages;
+
+/// <summary>
+/// Page model class for the dashboard page
+/// </summary>
+public class DashboardPage : ApplicationPage
+{
+    private WindowsElement AddWidgetButton => Driver.FindElementByAccessibilityId("AddWidgetButton");
+
+    private IReadOnlyCollection<WindowsElement> WidgetItems => Driver.FindElementsByAccessibilityId("WidgetItem");
```

Comment:
So we can have multiple elements with the same ID.. Is there no other field we could use? ID to me implies unique, though maybe this is standard practice for this

Reply:
`FindElementsByAccessibilityId` returns all the matches. If necessary, we can explore more options as part of this issue #1310